currency_rate_update/tests/test_currency_rate_update.py

```diff
@@ -41,7 +41,10 @@ def _test_cron_by_service(self, service_code, currency_xml_ids):
             rates.unlink()
         self.service_env._run_currency_update()
         logging.info("Service note: %s", service_x.note)
-        self.assertFalse('ERROR' in service_x.note)
+        if service_x.note:
+            self.assertFalse('ERROR' in service_x.note)
+        else:
+            logging.info('warning: No value returned from the service.')
```

Comment:
In a real deployment, I think we do want this at warning level. Can you check for the testing flag (see https://github.com/odoo/odoo/blob/10.0/addons/mail/models/res_partner.py#L237 for an example) and log as warning when this is not a test, and info otherwise?

Reply:
I think it's a good idea. I'll do some works and let you know.

Reply:
Any update on this?